recipes/libnfs/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conans import CMake, tools
+from conan import ConanFile
+import os
+import functools
+
+required_conan_version = ">=1.45.0"
+
+class LibnfsConan(ConanFile):
+    name = "libnfs"
+    description = "LIBNFS is a client library for accessing NFS shares over a network."
+    topics = ("async", "nfsv4", "nfs")
+    license = "LGPL 2.1"
```

Comment:
```suggestion
    license = "LGPL-2.1"
```

For the search engine, white space means different licenses/words